specification/app/resource-manager/Microsoft.App/preview/2022-11-01-preview/ContainerAppsJobs.json

```diff
@@ -0,0 +1,661 @@
+              "type": "string"
+            },
+            "type": "array"
+          },
+          "provisioningState": {
+            "description": "Provisioning state of the Container Apps Job.",
+            "enum": [
+              "InProgress",
+              "Succeeded",
+              "Failed",
+              "Canceled",
+              "Deleting"
+            ],
+            "type": "string",
+            "readOnly": true,
+            "x-ms-enum": {
+              "name": "ContainerAppsJobProvisioningState",
+              "modelAsString": true
+            }
+          },
+          "managedEnvironmentId": {
+            "description": "Deprecated. Resource ID of the Container App's environment.",
+            "type": "string",
+            "x-ms-mutability": [
+              "create",
+              "read"
+            ]
+          },
+          "environmentId": {
+            "description": "Resource ID of environment.",
+            "type": "string",
+            "x-ms-mutability": [
+              "create",
+              "read"
+            ]
+          },
+          "configuration": {
+            "$ref": "#/definitions/Configuration",
+            "description": "Non versioned Container Apps Job configuration properties."
+          },
+          "template": {
+            "$ref": "./CommonDefinitions.json#/definitions/ContainerAppsJobTemplate",
+            "description": "Container App versioned application definition."
+          },
+          "x-ms-client-flatten": true
+        }
+      }
+    },
+    "ContainerAppsJobsCollection": {
+      "description": "Container Apps Jobs collection ARM resource.",
+      "required": [
+        "value"
+      ],
+      "type": "object",
+      "properties": {
+        "value": {
+          "description": "Collection of resources.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ContainerAppsJob"
+          }
+        },
+        "nextLink": {
+          "description": "Link to next page of resources.",
+          "type": "string",
+          "readOnly": true
+        }
+      }
+    },
+    "RegistryCredentials": {
+      "description": "Container App Private Registry",
+      "type": "object",
+      "properties": {
+        "server": {
+          "description": "Container Registry Server",
+          "type": "string"
+        },
+        "username": {
+          "description": "Container Registry Username",
+          "type": "string"
+        },
+        "passwordSecretRef": {
+          "description": "The name of the Secret that contains the registry login password",
+          "type": "string"
+        },
+        "identity": {
+          "description": "A Managed Identity to use to authenticate with Azure Container Registry. For user-assigned identities, use the full user-assigned identity Resource ID. For system-assigned identities, use 'system'",
+          "type": "string"
+        }
+      }
+    },
+    "ContainerAppSecret": {
```

Comment:
ContainerAppJobSecret? although if this is similar to the ContainerAppSecret we cna try to reuse it here